mitosheet/src/utils/actions.tsx

```diff
@@ -576,8 +576,30 @@ export const createActions = (
             searchTerms: ['help', 'contact', 'support'],
             tooltip: "Join our Discord for more help."
         },
-        [ActionEnum.Import]: {
-            type: ActionEnum.Import,
+        [ActionEnum.Import_Dropdown]: {
+            type: ActionEnum.Import_Dropdown,
+            shortTitle: 'Import',
+            longTitle: 'Open import dropdown',
+            actionFunction: () => {
+                // We turn off editing mode, if it is on
+                setEditorState(undefined);
+
+                // we close the editing taskpane if its open
+                closeOpenEditingPopups();
+
+                setUIState(prevUIState => {
+                    return {
+                        ...prevUIState,
+                        displayImportToolbarDropdown: true
+                    }
+                })
+            },
+            isDisabled: () => {return undefined},
+            searchTerms: ['import', 'upload', 'new', 'excel', 'csv', 'add'],
```

Comment:
Instead of having an action to open the import dropdown, we'd be better off just having one to to open each import taskpane